cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -3,7 +3,7 @@ kind: ReplicationController
 metadata:
   # Keep the name in sync with image version and
   # gce/coreos/kube-manifests/addons/dashboard counterparts
-  name: kubernetes-dashboard-v1.0.0
+  name: kubernetes-dashboard-v1.0.1
   namespace: kube-system
   labels:
     k8s-app: kubernetes-dashboard
```

Comment:
Can you add a version label here (and below) and set it to 1.0.1 (or v1.0.1)? This will make the labeling consistent with the [dns addon](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in), and I believe that a change in labels in required for the updater to actually use the new yaml file to replace the running RC when upgrading a cluster. 

@ihmccreery @zmerlynn